### PR TITLE
Re-enable server error/warning scanning

### PR DESCRIPTION
**Pull request by aguibert**

### CODE REVIEW

**Comment by aguibert** [comment]
#build

**Comment by aguibert** [comment]
#build

**Comment by aguibert** [comment]
#build

**Comment by tbitonti** [comment]
Note: The very last comment is re: Iteratively building the buffer from the errorsInLog collection, not the 'toString' call.:

***            for (String errorInLog : errorsInLogs) {
***                sb.append("\n <br>");
***                sb.append(errorInLog);
                Log.info(c, method, "Error/warning found: " + errorInLog);
            }
            ex = new Exception(sb.toString());

**Comment by aguibert** [comment]
@tbitonti the code you've commended on was not functionally altered in my PR -- I just removed an extraneous if-statement that had been skipping over all of this code, and reduced the indentation.  However, I've removed the extra null check and also changed from StringBuffer to StringBuilder.  I wasn't sure how to resolve the buffer overflow issue you mentioned, so I've left that one alone.

**Comment by aguibert** [comment]
@tbitonti I've added the catch for the buffer overflow, but again, the code you are commenting on is outside of the scope of this code review.  The diff does make it look like I'm adding this code to LibertyServer, but the code has already been in place for a long time.

File: dev/com.ibm.ws.jaxrs.2.1.sse_fat/fat/src/com/ibm/ws/jaxrs21/sse/fat/SseJsonbTest.java [hunk 1]
```diff
@@ -54,7 +54,7 @@ public static void setUp() throws Exception {
 
     @AfterClass
     public static void tearDown() throws Exception {
-        server.stopServer();
+        server.stopServer("CWWKE1102W");
```
Comment (njr-11) [inline dev/com.ibm.ws.jaxrs.2.1.sse_fat/fat/src/com/ibm/ws/jaxrs21/sse/fat/SseJsonbTest.java]: Future TODO, check with JAX-RS team if this ought to really be ignored

**Pull request merged**